course4/homework-unique-pointer/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
Good; much more portable that the manual param tweaking I tend to use. If only there were a way to enable warnings the same way...